src/go/rpk/pkg/cli/cmd/wasm/common.go

```diff
@@ -105,3 +113,21 @@ func CreateRemoveMsg(name string) sarama.ProducerMessage {
 		Headers: headers,
 	}
 }
+
+func AddTypeFlag(command *cobra.Command, coprocType *string) {
+	command.Flags().StringVar(
+		coprocType,
+		"type",
+		"async",
+		"WASM engine type (async, data-policy)",
```

Comment:
Where do we have documentation on what these two engine types are? As a user, if I see this in just the flag help text, I'll know nothing.

Reply:
We have https://vectorized.io/blog/wasm-architecture/ for wasm engines.
But it is new feature, I do not know the right pipeline for that.

Reply:
make sure its documented in this PR. I left feedback about that in my last review